apps/mobile/src/features/token/token.tsx

```diff
@@ -86,7 +86,7 @@ export function Token({
               }
               actionButtons={
                 <ActionButtons
-                  onSend={() => sendSheetRef.current?.present()}
+                  onSend={() => asset && sendSheetRef.current?.present(asset)}
                   onReceive={() => receiveType && receiveSheetRef.current?.present(receiveType)}
                   canSend={canSend}
```

Comment:
Right now `canSend` is not used but I will work on adding `TokenDetails` for **Runes** soon and will use it for that